src/Tribe/RSVP.php

```diff
@@ -61,6 +61,13 @@ class Tribe__Tickets__RSVP extends Tribe__Tickets__Tickets {
 	public $security_code = '_tribe_rsvp_security_code';
 
 	/**
+	 * Meta key that if this attendee wants to show on the attendee list
+	 *
+	 * @var string
+	 */
+	public $optout = '_tribe_rsvp_attendee_optout';
```

Comment:
Hmm ok, it _is_ a bit subjective i guess. Do as you think best, but I do think this would be better as a class constant than a public property :pray: